src/config/feature-flags.ts

```diff
@@ -88,6 +89,23 @@ export const onFlagChange = (flag: BooleanFlags | StringFlags | NumberFlags, lis
 	launchdarklyClient.on(`update:${flag}`, listener);
 };
 
+type ShouldSendAllStringTypes =
+	"branches-backfill" | "builds-backfill" | "commits-backfill" | "deployments-backfill" | "prs-backfill" |
+	"branches" | "builds" | "commits" | "deployments" | "prs";
+
+export const shouldSendAll = async (type: ShouldSendAllStringTypes, jiraHost?: string, logger?: Logger): Promise<boolean> => {
```

Comment:
nit: curious why jiraHost and logger are optional?

Reply:
Cause some of the callers don't have it and lifting the values to that point would be quite hard from what I saw :|

I could be wrong tho! happy to be corrected here

Reply:
I quickly checked the callers: looks like there's only one that doesn't have logger around, but I couldn't spot any missing jiraHosts... :susp:

Reply:
This is the one I found: https://github.com/atlassian/github-for-jira/pull/2388/files#diff-e2e6b06c457708ee48dd963356ef9816859ade5936ff382cbb65a13c59639cb8L95

```
export const transformPullRequestRest = async (
	gitHubInstallationClient: GitHubInstallationClient,
	pullRequest: Octokit.PullsGetResponse,
	reviews?: Array<{ state?: string, user: Octokit.PullsUpdateResponseRequestedReviewersItem }>,
	log?: Logger,
	jiraHost?: string // <<<< ----------
)
```

Reply:
@tavomaciel https://github.com/atlassian/github-for-jira/pull/2393/files - that should fix your problem, please feel free to merge in the branch :)